reactioncommerce/reaction-platform
# Chore/docs

## Issue
links for docs are outdated (in readme and github templates)

## Solution
remove 404s and pointed to current docs and discord as needed

## Breaking changes
none


## Testing
1.  open a pull request and review default links
2. check readme
